lib/crypt_utils.rb

```diff
@@ -109,6 +109,30 @@ def decrypt_blob(blob, team_password)
       decrypted_blob << cipher.final
       decrypted_blob.force_encoding('UTF-8')
     end
-  end
 
+    def encrypt_base64(data, cipher_key, iv)
```

Comment:
here you called it cipher_key ... further up, it's called encryption_key. 

I prefer encryption_key over cipher_key ... or just key ?